openedx_webhooks/tasks/pr_tracking.py

```diff
@@ -263,6 +263,7 @@ def desired_support_state(pr: PrDict) -> Optional[PrDesiredInfo]:
     if not has_signed_agreement:
         desired.bot_comments.add(BotComment.NEED_CLA)
         desired.jira_initial_status = "Community Manager Review"
+        desired.github_labels.add('NEED-CLA')
```

Comment:
Looking at this again, I think I need to update the labels earlier on in the helper, maybe somewhere around `self._fix_github_labels()` :wink: